static/templates/right_sidebar.hbs

```diff
@@ -1,5 +1,13 @@
 <div class="right-sidebar" id="right-sidebar" role="navigation">
     <div class="right-sidebar-items">
+        {{#if page_params.is_spectator }}
+        <p>{{ page_params.realm_description }}</p>
+        <p><a href="/help/anonymous-users">Checkout help on Anonymous Users</a></p>
+        <div class="login_button color_animated_button">
+            <i class="fa fa-sign-in"></i>
+            <span>Log in</span>
+        </div>
+        {{/if}}
```

Comment:
It looks like this gets removed in favor of hiding the sidebar in a later commit; does it make sense to just squash these such that this commit disappears?

Reply:
I had planned to drop the commit after you looked at the UI with right sidebar removed. Dropped now.